### PR TITLE
Update README.md version to 0.5.1

### PR DESCRIPTION
**Pull request by abchoo**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ generation [script](./hack/self-signed-ca.sh).
 KFServing can be installed standalone if your kubernetes cluster meets the above prerequisites and KFServing controller is deployed in `kfserving-system` namespace.
 
 ```
-TAG=v0.5.0
+TAG=v0.5.1
 ```
 
 Install KFServing CRD
```
Comment (pvaneck) [inline README.md:60]: In v0.5.1, the crd yaml and controller yaml are now in the same file (`kfserving.yaml`), so the instructions below need to be changed by removing the instruction to apply the `kfserving_crds.yaml` file.

**Review by pvaneck — COMMENTED** [review]
One comment regarding the installation instructions.

File: README.md (hunk 1 above)
Comment (abchoo) [reply README.md]: Great catch!

**Review by pvaneck — COMMENTED** [review]
/lgtm

**Pull request merged**